# Output from DNN CV pose estimation

Header header

# Drone position
bool position_available # indicates whether or not the message contains a valid position estimate
float64 x_n
float64 y_n
float64 z_n

# Yaw orientation
bool heading_available # indicates whether or not the message contains a valid heading estimate
float64 psi